app/car.py

```diff
@@ -0,0 +1,11 @@
+class Car:
+    def __init__(
+            self,
+            brand: str,
+            fuel_consumption: int | float
+    ) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    def one_km_consumption(self) -> float:
```

Comment:
Either make this method as property or include the verb in the name, e.g.  `get_one_km_consumption` to represent action